lib/Runtime/Language/i386/AsmJsJitTemplate.cpp

```diff
@@ -1507,6 +1507,16 @@ namespace Js
             int32 stackSize = asmInfo->GetTotalSizeinBytes();
             stackSize = ::Math::Align<int32>(stackSize, 8);
 
+#if DBG_DUMP
+            if (Js::Configuration::Global.flags.DebugBreak.Contains(funcBody->GetFunctionNumber()) ||
+                PHASE_ON(DebugBreakPhase, funcBody))
+            {
+                // int 3
+                *buffer++ = 0xCC;
```

Comment:
I thought about it, but I don't think it's worth it since we will not use int instruction much.
Also, 0xCC is a special case for `int 3` on x86
![image](https://cloud.githubusercontent.com/assets/4157103/25687762/04da1254-302e-11e7-8cc0-c83d6c0ad559.png)